client/app/telecom/dashboard/guides/telecom-dashboard-guides.html

```diff
@@ -8,7 +8,8 @@
                title="{{ ::'telecom_dashboard_FAQ_Activate' | translate }} {{ ::'common_newtab' | translate }}"
                target="_blank"
                rel="noopener"
-               data-ng-href="{{ GuidesCtrl.links.packActivate }}">
+               data-ng-href="{{ GuidesCtrl.links.packActivate }}"
+               data-ng-click="GuidesCtrl.trackRedirection(GuidesCtrl.links.packActivate)">
```

Comment:
I think you will have some side effet with this technic. ref https://github.com/angular/angular.js/issues/6671.